src/app/containers/IndexHeading/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { node } from 'prop-types';
+import IndexHeading from '@bbc/psammead-heading-index';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+  GEL_GROUP_5_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING, GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const StyledIndexHeading = styled(IndexHeading)`
+  padding-bottom: ${GEL_SPACING_TRPL};
```

Comment:
All these paddings will have to be reviewed once we take this component out of alpha and reuse it in the Most Read page since now there are taking into account the margins applied to the first section to match the IDX UX designs.